frontend/public/components/factory/Table/VirtualizedTable.tsx

```diff
@@ -52,6 +52,20 @@ const isColumnVisible = <D extends any>(
   return true;
 };
 
+export type TableDataProps = {
+  id: string;
+  activeColumnIDs: Set<string>;
+  className?: string;
+};
+
+export const TableData: React.FC<TableDataProps> = ({ className, id, activeColumnIDs, children }) =>
```

Comment:
this component replaces https://github.com/openshift/console/blob/master/frontend/public/components/factory/table.tsx#L217

the old one has some additional logic (`isColumnVisible`) which is not needed anymore as new Rows get a list of active columns so we just need to check if the `id` is there.

This new component also has less props & will be exposed as part of the dynamic plugin API